clap_complete/src/engine/custom.rs

```diff
@@ -274,7 +274,19 @@ impl ValueCompleter for PathCompleter {
             current_dir_actual = std::env::current_dir().ok();
             current_dir_actual.as_deref()
         });
-        let mut candidates = complete_path(current, current_dir, filter);
+
+        let Some(current) = current.to_str() else {
+            return vec![];
+        };
+
+        #[cfg(unix)]
+        if current.starts_with("~") {
```

Comment:
Technically, there are other variants of `~` expansion.  Should we support that?

Reply:
I'm not sure about what they are, can you explain?